webpack.config.common.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+    resolve: {
+        extensions: ['.js', '.jsx']
+    },
+    entry: './src/scripts/main.js',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: '[name].js',
+    },
+    plugins: [
+        new webpack.HotModuleReplacementPlugin(),
```

Comment:
do we really need `new webpack.HotModuleReplacementPlugin(),` for prod config as well?

We do benefit from it, but only for development purposes as it said in official doc: https://webpack.js.org/concepts/hot-module-replacement/